chainmanager/README.md

```diff
@@ -0,0 +1,28 @@
+# Chainmanager Module
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Query commands](#query-commands)
+
+## Overview
+
+The chainmanager module is responsible for fetching the chainmanager params. These params include contract address of mainchain and maticchain, chain ids, mainchain and maticchain confirmation blocks
```

Comment:
Just to be clear, should we elaborate a bit more on what `mainchain` and `maticchain` mean here ?

Reply:
updated. Please check

Reply:
Also a note for the future, we should probably replace the term `maticchain` with `borchain` or something. Thoughts @maticnetwork/pos-v1 ?

Reply:
> updated. Please check
LGTM , thanks!

Reply:
@Raneet10 we can continue this discussion on slack for better responses. Merging this for now

Reply:
Sure, sure.